annotator/src/test/java/org/cbioportal/annotator/internal/GenomeNexusImplTest.java

```diff
@@ -186,6 +186,33 @@ public void testPostVsGetAnnotationResults() throws Exception {
         }
     }
 
+    @Test
+    public void testGnomadAnnotationResultsExcluded() throws Exception {
+        // test records that should not have the gnomad records
+        for (AnnotatedRecord record : mockAnnotatedRecordsWithPost) {
+            if (record.getGNOMAD_AF() != null) {
+                Assert.fail("testGnomadAnnotationResultsExcluded(), annotated record includes GNOMAD_AF when it should not, value is: '" + record.getGNOMAD_AF() + "'");
+            }
+        }
+    }
+
+    @Test
+    public void testGnomadAnnotationResultsIncluded() throws Exception {
+        // annotate now with my_variant_info included in enrichmentFields
+        ReflectionTestUtils.setField(annotator, "enrichmentFields", GenomeNexusTestConfiguration.MY_VARIANT_INFO_ENRICHMENT_FIELDS);
```

Comment:
Here we are manually injecting a setting into the annotator. This probably indicates that too much was hardcoded into the annotator. Perhaps in the future we will remove the hardcoding and allow an argument to determine the enrichmentFields to be used for each annotation call.